DigitalLearningSolutions.Data/Helpers/CourseCategoryHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers
+{
+    public static class CourseCategoryHelper
+    {
+        /// <summary>
+        ///     Converts a category ID into a category filter to match the
+        ///     data service convention of not filtering on NULL category filter
+        /// </summary>
+        /// <param name="categoryId"></param>
+        /// <returns></returns>
+        public static int? GetCourseCategoryFilter(int categoryId)

```

Comment:
After some discussion in the relevant thread in slack, I've decided to get rid of the helper, and just incorporate the logic into  the custom claim method. Let me know if you think it would be better elsewhere.